www/content/docs.md

```diff
@@ -1028,13 +1028,13 @@ htmx includes a number of useful headers in requests:
 | `HX-Trigger-Name` | the `name` of the triggered element if it exists
 | `HX-Trigger` | the `id` of the triggered element if it exists
 
-### Response Headers
+### Response Headers {#response_headers}
```

Comment:
This seems to break the default anchor that was `response-headers` and referenced at 2 other places in this file.
Ran `zola serve` and got 2 errors, I'd say either replace this by `response-headers` or replace the 2 existing usages by `response_headers`.

Reply:
Yeah sorry I missed those two existing links.  should be fixed now